pkg/auth/service.go

```diff
@@ -1264,6 +1267,7 @@ func userIDToInt(userID string) (int64, error) {
 }
 
 func (a *APIAuthService) getFirstUser(ctx context.Context, userKey userKey, params *ListUsersParams) (*model.User, error) {
+	ctx = httputil.SetClientTrace(ctx, "api_auth")
```

Comment:
This is a private method, I believe you already added this in the callers

Reply:
Done.